src/main.cpp

```diff
@@ -310,11 +322,26 @@ int main(int argc, char * argv[])
     if (!(ErrorDialogHandler::instance()->checkError())) {
         qDebug() << "Displaying mixxx";
         mixxx->show();
+        splash.finish(mixxx);
 
         qDebug() << "Running Mixxx";
         result = a.exec();
     }
 
+    QMessageBox cleaningUpBox;
+    cleaningUpBox.setStandardButtons(0);
+    cleaningUpBox.setWindowTitle(QObject::tr("Exiting Mixxx"));
+    cleaningUpBox.setText(QObject::tr(
+            "Cleaning up the library. This may take some time.\n"
+            "Do not restart Mixxx or shutdown the device until\n "
+            "this box has disappeared."));
```

Comment:
Can you propose an improved Text? 

There is nothing stated about the OS cache and disk writes. This is a separate issue. 

I think we have these issues, if Mixxx is shutdown early: 
* Played flags counter may stick
* Metadata changes might be lost. 
* Infos stored in journal files are not integrated to the presistant file. 

We have these issues, if Mixxx is restarted before the old instants is gone: 
* Audio Driver is not available
* Database race conditions